Task3/MoneyManager/MoneyManager.Business/Services/MixedService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.NetworkInformation;
+using MoneyManager.Business.Models;
+using MoneyManager.DataAccess.Models;
+using MoneyManager.DataAccess.Repositories;
+
+namespace MoneyManager.Business.Services
+{
+    public class MixedService
```

Comment:
`MixedService` is not good name for service. I think you should move method from this service to `CategoryService` and remove this file.